compiler/src/language_server/protocol.rei

```diff
@@ -14,7 +15,7 @@ type version;
 type message_id;
 
 [@deriving yojson]
-type uri;
+type uri = Uri.t;
```

Comment:
This leaks the implementation of the type, but I believe it's the easier way to get this to work with the separate utils file.

Reply:
Yes, it's the only way I could find, as the types didn't match otherwise when calling the Uri functions.